.config-schema.json

```diff
@@ -120,6 +120,17 @@
             "default": [],
             "x-env-variable": "OPENFGA_EXPERIMENTALS"
         },
+        "checkTrackerEnabled": {
+            "type": "object",
+            "properties": {
+                "enabled": {
+                    "description": "Enable/disable logging the number of hits on an edge .",
```

Comment:
As I asked in https://github.com/openfga/openfga/pull/1745#discussion_r1674760708, can you please update this description? Here and in this line: `flags.Bool("check-tracker-enabled", defaultConfig.CheckTrackerEnabled, `enabled/disabled logging the number of hits on an edge`)`.

People using OpenFGA have no context around what this flag means unless we tell them :)